docs/develop/secrets.md

```diff
@@ -1,9 +1,123 @@
 # Secret management
 
-{% hint style="warning" %}
+Some models need access to APIs, databases, AWS resources, or other secured information. With Truss, you can securely reference access keys, API tokens, passwords, secrets, and more. Keep this sensitive information out of your Git repository and use the same best practices for ML models as production websites.
```

Comment:
not sure but perhaps `production services` may be more relevant to our target audience compared to `production websites`.